patches/0001-Add-flatpak-ed-Steam-data-dir.patch

```diff
@@ -0,0 +1,30 @@
+From fd4e167f485efde5e3828f44baf697ea480c6a65 Mon Sep 17 00:00:00 2001
+From: gasinvein <gasinvein@gmail.com>
+Date: Sun, 24 Mar 2019 17:02:10 +0300
+Subject: [PATCH 1/2] Add flatpak'ed Steam data dir
+
+---
+ lutris/runners/steam.py | 7 ++++++-
+ 1 file changed, 6 insertions(+), 1 deletion(-)
+
+diff --git a/lutris/runners/steam.py b/lutris/runners/steam.py
+index dbc9ed56..e0a1b1e4 100644
+--- a/lutris/runners/steam.py
++++ b/lutris/runners/steam.py
+@@ -174,7 +174,12 @@ class steam(Runner):
+     @property
+     def steam_data_dir(self):
+         """Return dir where Steam files lie."""
+-        candidates = ("~/.steam", "~/.local/share/steam", "~/.steam/steam")
++        candidates = (
++            "~/.steam",
++            "~/.local/share/steam",
++            "~/.steam/steam",
++            "~/.var/app/com.valvesoftware.Steam/data/steam",
```

Comment:
Any thoughts on pushing this change upstream?

Reply:
Not currently, I think. It's useless without the second patch, and the second patch should be done better, probably avoiding calling `xdg-open`.